src/_designs/components/labelType/index.vue

```diff
@@ -31,10 +31,11 @@ export default {
   @import "~@aeternity/aepp-components-3/src/styles/variables/colors";
   .label-wrapper {
     display: table-caption;
-    padding: 5px;
+    padding: 0.6rem;
     text-transform: uppercase;
     color: white;
     border-radius: 0 5px 5px 0;
+    font-weight: bold;
   }
   .label-wrapper.red {
     background-color: red;
```

Comment:
we need the right hex colors here. Not just CSS defaults.

Reply:
done